api/gen/es/rpc_connect.d.ts

```diff
@@ -173,7 +173,9 @@ export declare const GnoNativeService: {
       readonly kind: MethodKind.Unary,
     },
     /**
-     * SelectAccount selects the active account to use for later operations
+     * SelectAccount selects the active account to use for later operations. If the response has_password is
+     * false, then you should set the password before using a method which needs it.
+     * If the key doesn't exist, return [ErrCode](#land.gno.gnomobile.v1.ErrCode).ErrCryptoKeyNotFound.
```

Comment:
Why `gnomobile` and not `gnonative`?

Reply:
Thanks for catching this. I committed a fix. (I started working on the PR a long time ago ;-)